### DISCUSSION TITLE
Congrats on enforcing a product that doesn't even work

### DISCUSSION THREAD

**Discussion by Naville** [community/community#66253] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="1009" alt="image" src="https://github.com/community/community/assets/5205699/028f473f-0473-43dd-8902-6f3a77812cb3">
Apparently no matter how I poke with the filter settings I could only see my new followers in the stream.

**Comment by sgf** [comment]
The homepage of the new version is very confusing, and I can't see which valuable repositories have been starred by the people I follow.
I want to go back to the old home page.

**Comment by Naville** [comment]
After a "Reset to Defaults", it now only shows Releases from the projects I follow:

<img width="1067" alt="image" src="https://github.com/community/community/assets/5205699/66d0f067-5054-485f-ad3c-961e78572b80">

**Comment by loneicewolf** [comment]
Same here. Although I rarely make a discussion with such a title, this should have been tested. And I wonder when it will be corrected ? 🤔

**Reply by loneicewolf** [reply]
oh btw as I don't follow anyone (I am aware of this) I replied to the other parts of this discussion :)
